opensrp-chw/src/chad/assets/json.form/pnc_health_facility_visit.json

```diff
@@ -167,6 +167,13 @@
         "read_only": "true",
         "hidden": "true",
         "value": "0"
+      },
+      {
+        "key": "pnc_hf_next_visit_date",
```

Comment:
The same issue looks like a copy-paste on all the other fields

Reply:
Fixed as mentioned above